src/Mod/Draft/draftobjects/dimensionstyle.py

```diff
@@ -28,13 +28,12 @@
 # \brief This module provides the object code for Draft DimensionStyle.
 
 import FreeCAD as App
-import Draft
-from Draft import _DraftObject
+from draftobjects.draft_annotation import DraftAnnotation
 from PySide.QtCore import QT_TRANSLATE_NOOP
 
 if App.GuiUp:
     import FreeCADGui as Gui
-    from draftviewproviders.view_style_dimension import ViewProviderDraftDimensionStyle
+    from draftviewproviders.view_dimensionstyle import ViewProviderDraftDimensionStyle
```

Comment:
I don't think we need to guard the view providers because they won't be executed by the import, so there is no risk of causing an error. I mean, it doesn't hurt at all to guard them like this, I just think it looks a bit ugly.

We have to guard `FreeCADGui` because if we don't we will definitely cause an error in console mode.

Reply:
About this, I think some Travis tests are failing when we import the view provider module because inside this module `FreeCADGui` is not guarded.

So inside `view_dimensionstyle.py` and all those new view providers.
```py
if App.GuiUp:
    import FreeCADGui as Gui
```